openfe/protocols/openmm_md/plain_md_methods.py

```diff
@@ -326,38 +326,39 @@ def _run_MD(simulation: openmm.app.Simulation,
         )
         # equilibrate
         # NVT equilibration
-
-        if verbose:
-            logger.info("Running NVT equilibration")
-
-        # Set barostat frequency to zero for NVT
-        for x in simulation.context.getSystem().getForces():
-            if x.getName() == 'MonteCarloBarostat':
-                x.setFrequency(0)
-
-        simulation.context.setVelocitiesToTemperature(
-            to_openmm(temperature))
-
-        t0 = time.time()
-        simulation.step(equil_steps_nvt)
-        t1 = time.time()
-        if verbose:
-            logger.info(
-                f"Completed NVT equilibration in {t1 - t0} seconds")
-
-        # Save last frame NVT equilibration
-        positions = to_openmm(
-            from_openmm(simulation.context.getState(
-                getPositions=True, enforcePeriodicBox=False
-            ).getPositions()))
-
-        traj = mdtraj.Trajectory(
-            positions[selection_indices, :],
-            mdtraj_top.subset(selection_indices),
-        )
-        traj.save_pdb(
-            shared_basepath / output_settings.equil_NVT_structure
-        )
+        if equil_steps_nvt is not None or equil_steps_nvt != 0:
```

Comment:
I think this can just be `if equil_steps_nvt:`.  You've got three possible cases, None, 0 and >0, and that if statement will correctly work on those

Reply:
Changed this!